i18n-embed/src/assets.rs

```diff
@@ -66,43 +66,48 @@ impl I18nAssets for FileSystemAssets {
             Ok(contents) => Some(Cow::from(contents)),
             Err(e) => {
                 log::error!(
-                    target: "i18n_embed::assets", 
-                    "Unexpected error while reading localization asset file: {}", 
+                    target: "i18n_embed::assets",
+                    "Unexpected error while reading localization asset file: {}",
                     e);
                 None
             }
         }
     }
 
     fn filenames_iter(&self) -> Box<dyn Iterator<Item = String>> {
-        Box::new(
-            walkdir::WalkDir::new(&self.base_dir)
-                .into_iter()
-                .filter_map(|f| match f {
-                    Ok(f) => {
-                        if f.file_type().is_file() {
-                            match f.file_name().to_str() {
-                                Some(filename) => Some(filename.to_string()),
-                                None => {
-                                    log::error!(
-                                    target: "i18n_embed::assets", 
-                                    "Filename {:?} is not valid UTF-8.", 
+        match globwalk::GlobWalkerBuilder::new(&self.base_dir, "*").build() {
```

Comment:
Perhaps we can leave the walkdir dependency here because we aren't yet using any globwalk features from what I can tell, and it is already an internal dependency of globwalk?